shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -919,12 +920,12 @@ verifiedWits (Tx txbody wits _) =
     failed =
       wvkKey
         <$> filter
-          (not . verifyWitVKey (coerce . hashAnnotated $ txbody))
+          (not . verifyWitVKey (extractHash (hashAnnotated @(Crypto era) txbody)))
```

Comment:
getting rid of coerce is a good consequence

Reply:
I agree